modules/rules_swift_package_manager/0.5.0/MODULE.bazel

```diff
@@ -0,0 +1,77 @@
+module(
+    name = "rules_swift_package_manager",
+    version = "0.5.0",
+)
+
+# MARK: - Runtime Dependencies
+
+bazel_dep(name = "cgrindel_bazel_starlib", version = "0.15.0")
+bazel_dep(name = "bazel_skylib", version = "1.4.1")
+bazel_dep(
+    name = "rules_go",
+    version = "0.38.1",
+    repo_name = "io_bazel_rules_go",
+)
+bazel_dep(name = "rules_cc", version = "0.0.6")
+bazel_dep(name = "platforms", version = "0.0.6")
+
+# With bzlmod enabled, any dependencies for repositories generated by
+# rules_swift_package_manager _must_ be a runtime dependency of rules_swift_package_manager.
+bazel_dep(
+    name = "rules_swift",
+    version = "1.6.0",
+    repo_name = "build_bazel_rules_swift",
+)
+bazel_dep(
+    name = "rules_apple",
+    version = "2.1.0",
+    repo_name = "build_bazel_rules_apple",
+)
+
+# TODO(chuck): FIX ME! Gazelle needs a new release to get the following fix:
+# https://github.com/bazelbuild/bazel-gazelle/pull/1413
+bazel_dep(
+    name = "gazelle",
+    version = "0.29.0",
+    repo_name = "bazel_gazelle",
+)
+
+go_sdk = use_extension("@io_bazel_rules_go//go:extensions.bzl", "go_sdk")
+use_repo(
+    go_sdk,
+    go_sdk = "go_default_sdk",
```

Comment:
We would like to remove downstream usages of this repo so that it isn't fetched as often. Is everything in place for you to migrate to `@rules_go//go`?

Reply:
(@cgrindel this is of course not a blocker for this PR, just merged it)

Reply:
I just created cgrindel/rules_swift_package_manager#408 to track it. I should be able to look into it in the next day or so.